This fixes:

../../ui/display/types/display_color_management.h:36:8: error: no type named 'string' in namespace 'std'
  std::string ToString() const;
  ~~~~~^

Upstream included it in a much larger patch, but we just need the header.

--- a/ui/display/types/display_color_management.h
+++ b/ui/display/types/display_color_management.h
@@ -5,6 +5,7 @@
 #ifndef UI_DISPLAY_TYPES_DISPLAY_COLOR_MANAGEMENT_H_
 #define UI_DISPLAY_TYPES_DISPLAY_COLOR_MANAGEMENT_H_
 
+#include <memory>
 #include <vector>
 
 #include "third_party/skia/modules/skcms/skcms.h"
